### PR TITLE
Jesse/issue 1

### PR DESCRIPTION
**Pull request by vc-ca**
UI is complete.

For the login page I kept the username field rather than changing to email. Changing to email would have required reworking the non UI code.

For the sign up page, I kept in the second confirmation password field for create new user. I believe it's better UX, for a new user to have to repeat their PW in case they enter it incorrectly. 

View on Desktop:
image

View on Mobile:
image

### CODE REVIEW

File: client/src/Login.js [hunk 1]
```diff
@@ -28,40 +80,196 @@ const Login = (props) => {
...
+                align="left"
+                variant="h4"
+                style={{ marginTop: "86px" }}
+              >
+                Welcome Back!
+              </Typography>
+            </Grid>
+            <Grid item xs={12}>
+              <form onSubmit={handleLogin} autoComplete="off">
+                {/* <Grid xs={12} style={{ marginLeft: "16%", marginTop: "12%" }}> */}
```
Comment (RayK3) [inline client/src/Login.js:234]: Make sure to remove any commented out sections of code before submitting a PR 👍

File: client/src/Login.js [hunk 2]
```diff
@@ -28,40 +80,196 @@ const Login = (props) => {
...
+                <FormControl fullWidth margin="normal" required>
+                  <TextField
+                    required
+                    aria-label="username"
+                    label="Username"
+                    name="username"
+                    type="text"
+                    color="#3A8DFF"
```
Comment (RayK3) [inline client/src/Login.js:242]: This is a sneaky one, so I'll mention it separately. This colour is technically a style still and should be included with the `makeStyles` from above, and probably gotten from the theme as well.

File: client/src/Signup.js [hunk 3]
```diff
@@ -37,72 +89,239 @@ const Login = (props) => {
...
-            </Button>
           </Grid>
-        </form>
-      </Box>
+          <Grid item xs={2} />
+        </Grid>
+      </Grid>
     </Grid>
   );
 };
```
Comment (RayK3) [inline client/src/Signup.js:327]: All of the same changes to styling (using theme, moving all inline styles to class styles) should be repeated in this file 👍

**Review by RayK3 — COMMENTED** [review]
The pages look good, and nice job including such a descriptive README with pictures. I had some comments about using the Material-UI theme to get colours/padding/margin, as well as refactoring the styling away from inline styles. Since there is a lot of repeated code between the `Login` and `Signup` pages, I would encourage you to think about some potential ways of refactoring to reduce the duplication :grin:

File: client/src/Login.js [hunk 4]
```diff
@@ -3,15 +3,67 @@ import { Redirect, useHistory } from "react-router-dom";
...
+    backgroundSize: "cover",
+    backgroundPosition: "center",
+  },
+  welcomeImgMobile: {
+    backgroundImage: `linear-gradient(to bottom,  rgba(58, 141, 255, .85), rgba(134, 185, 255, .85)), url(${bgImg})`,
+    height: "25vh",
+    backgroundRepeat: "no-repeat",
+    backgroundSize: "cover",
+    backgroundPosition: "top",
+  },
```
[the remark this answers is not in the document]
Comment (vc-ca) [reply client/src/Login.js:31]: Noted!

File: client/src/Login.js [hunk 5]
```diff
@@ -3,15 +3,67 @@ import { Redirect, useHistory } from "react-router-dom";
...
+    width: "160px",
+    height: "56px",
+    marginTop: "60px",
+    marginBottom: "0",
+    marginLeft: "auto",
+    marginRight: "auto",
+    display: "flex",
+    borderRadius: "3px",
+  },
+});
```
[the remark this answers is not in the document]
Comment (vc-ca) [reply client/src/Login.js:63]: I hadn't worked much with theme, and will look into it

File: client/src/Login.js [hunk 6]
```diff
@@ -28,40 +80,196 @@ const Login = (props) => {
...
+          className={classes.welcomeImg}
+          container
+          item
+          sm={3}
+          md={5}
+          direction="row"
+          justify="center"
+          alignItems="flex-start"
+        >
+          <Grid container style={{ paddingLeft: "20%", paddingRight: "20%" }}>
```
[the remark this answers is not in the document]
Comment (vc-ca) [reply client/src/Login.js:97]: I agree, it's a bit messy.

File: client/src/Login.js (hunk 2 above)
Comment (vc-ca) [reply client/src/Login.js:242]: 👍

**Comment by vc-ca** [comment]
Yeah when building this I considered having a single separate css file with which sign-in & login could both draw from to reduce duplication. That would also reduce potential disparity between the styles

**Review by bonnieli — APPROVED** [review]

**Pull request merged**